### DISCUSSION TITLE
Is there a way to interpolate between two images?

### DISCUSSION THREAD

**Discussion by TinyDeskEngineer06** [dlemstra/Magick.NET#1637] [Help]
I'm trying to do something that requires me to create new images by linearly interpolating between the colors of two images (That may or may not have their own alpha channels, which will also need to be interpolated like the color channels). MagickImage.Compose is getting _very_ close with the Dissolve operator, but it doesn't seem to be quite what I need.

**Comment by dlemstra** [comment]
I am not sure what you want to do. Maybe you can ask your question here first: https://github.com/ImageMagick/ImageMagick/discussions. I can then help you translate this to C# if you need help with that.
